packages/web/hooks/one-click-trading/use-one-click-trading-info.ts

```diff
@@ -43,9 +82,31 @@ export const useOneClickTradingInfo = ({
     return () => clearTimeout(timeoutId);
   }, [isExpired, t, value?.info, onExpire]);
 
+  const getTimeRemaining = useCallback(() => {
```

Comment:
Since this takes no params, why don't we use a `useMemo` to just return the remaining time?

Reply:
Since the info object contains a hardcoded session expiry time, it won't update automatically, returning inaccurate data. We also don't want to add an interval on this hook since it would trigger unnecessary re-renders. Therefore, we return a function so the components can handle the update logic themselves as we do on `WalletInfo`, setting a `setInterval`